tests/property/ic_api/cycles/src/cycles/index.ts

```diff
@@ -62,7 +54,18 @@ function acceptCycles(
     };
 }
 
-function acceptCyclesRecursive(
+function acceptCyclesStrategy(
```

Comment:
Can we just have two functions? `acceptCyclesAll` and `acceptCyclesChunk`. Seems simpler

Reply:
I am not convinced that this is either simpler or more declarative, but if you like it here it is